extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.json

```diff
@@ -0,0 +1,172 @@
+              "type":"boolean"
+            },
+            "journey:reentrance":{
+              "title":"reentrance",
+              "description":"Indicate if the user has reentered the journey with the same instance. If not present, assuming that the value is false",
+              "type":"boolean"
+            },
+            "journey:instanceEnded":{
+              "title":"instanceEnded",
+              "description":"Indicate if the instance is ended (sucessfully or not)",
+              "type":"boolean"
+            },
+            "journey:eventId":{
+              "title":"eventId",
+              "description":"Event id in processing, for the step processing",
+              "type":"string"
+            },
+            "journey:nodeId":{
+              "title":"nodeId",
+              "description":"client node id",
+              "type":"string"
+            },
+            "journey:stepId":{
+              "title":"stepId",
+              "description":"Unique id of the step that is currently in processing",
+              "type":"string"
+            },
+            "journey:stepName":{
+              "title":"stepName",
+              "description":"Name of the step that is currently in processing",
+              "type":"string"
+            },
+            "journey:stepType":{
+              "title":"stepType",
+              "description":"Type of the step",
+              "type":"string"
+            },
+            "journey:stepStatus":{
+              "title":"stepStatus",
+              "description":"Status of the journey step, representing in which shape the step is, when its processing has been done",
+              "type":"string"
+            },
+            "journey:journeyId":{
+              "title":"journeyId",
+              "description":"ID of the journey",
+              "type":"string"
+            },
+            "journey:journeyVersionId":{
+              "title":"journeyVersionId",
+              "description":"ID of the journey version. This id represents the identity reference to the journey, in case of the journeyStepEvent",
+              "type":"string"
+            },
+            "journey:journeyVersionName":{
+              "title":"journeyVersionName",
+              "description":"Name of the journey version",
+              "type":"string"
+            },
+            "journey:journeyVersion":{
+              "title":"journeyVersion",
+              "description":"Version of the journey version",
+              "type":"string"
+            },
+            "journey:instanceId":{
+              "title":"instanceId",
+              "description":"Internal ID of the journey instance",
+              "type":"string"
+            },
+            "journey:externalKey":{
+              "title":"externalKey",
+              "description":"External key extracted from the event to process it",
+              "type":"string"
+            },
+            "journey:parentStepId":{
```

Comment:
why not simply have define all these fields (stepID, stepName, etc.) as their own schema and then have one reference to it for the journey and another for the parent?